app/lib/actions/pulp/orchestration/repository/remove_units.rb

```diff
@@ -0,0 +1,13 @@
+module Actions
+  module Pulp
+    module Orchestration
+      module Repository
+        class RemoveUnits < Pulp::Abstract
+          def plan(_repository, _smart_proxy, options)
+            plan_action(Actions::Pulp::Repository::RemoveUnits, options)
```

Comment:
can we get rid of the repo_id param going into RemoveUnits orchestration tasks, and use the repository being passed in?

Reply:
As long as we are ok refactoring pulp2 things, sure. I'm assuming that's what we're talking about here.

Reply:
I actually meant to leave the pulp2 action alone, but in this orchestration action, just do something like:
```
options[:repo_id] = repository.id
```

and remove the 'repo_id being passed in from the  RemoveContent action.  That way, the RemoveContent action doesn't have any pulp2-specific code, which this 'repo_id' param is.  Does that make sense?

Reply:
Oh! I see what you meant. Makes sense.